README.md

```diff
@@ -2,41 +2,22 @@
-
-### Option 2: How to *use as template*
-
-Go to https://github.com/SoCSTech/CMP9137 and click on 
-
-![usetemplate](.assets/usetemplate.png)
-
-to create a new repository in your own GitHub account. Then continue to [clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) this repository on your computer. *Note: This repository is detached from the original one, so any changes in https://github.com/SoCSTech/CMP9137 happening after you created your own repository will have to be manually added.
-
-
-### Setup your working environment
-
-1. Make sure you have VSCode installed: https://code.visualstudio.com/download
-2. Make sure you have the `Docker` and the `Dev Containers` extension in VSCode installed and working: https://code.visualstudio.com/docs/containers/overview and https://code.visualstudio.com/docs/devcontainers/containers
-    * ensure docker is working, i.e. try `docker run --rm hello-world` and check it succeeds for your user
-3. The docker image used to provide the Development Container is provided by the Docker Container Registry. You do not need to log in to use it.
-
-### Open in VSCode
-
-1. Open your own repository (*forked* or generated from this template) in VSCode: https://code.visualstudio.com/docs/sourcecontrol/intro-to-git (or any other way you prefer), e.g. click on "Clone Respository" in VSCode:
-    ![Alt text](.assets/clone.png)
-
-2. VSCode should prompt you that there is a devcontainer configured and ask if you want to reopen in container. Re-open in the container
\ No newline at end of file
+## Use Case
+
+You can use this repository use any version of Tensorflow on the lab machines, or at home.
+
+## How to Use
+
+1. Click the green 'Code' button and download as a zip.
+2. Extract the zip file to the desktop.
+3. Open the 'Docker Desktop' application. Once it has launched you may minimise the window.
+4. Go to the C:\Users\Computing\Desktop\CMP9137-main\CMP9137-main directory in File Explorer.
+5. By default, this container runs Tensorflow version 2.10.0. If you require a different version, open the .devcontainer folder, right click the devcontainer.json file, click 'Edit in Notepad', and locate line 5: '"image": "tensorflow/tensorflow:2.10.0-gpu",'. This line may be edited to a different Tensorflow version. For example, you may change it to: '"image": "tensorflow/tensorflow:2.15.0-gpu",'. Different versions of tensorflow, with their corresponding preferred CUDA versions can be found at www.tensorflow.org/install/source#gpu
+6. Right click in any blank space in the File Explorer window, and click 'Open in Terminal'
+7. In the terminal, type 'code .' and press enter.
```

Comment:
Add a step here to say "if you're working from home, you will need to install the [devcontainers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) for it to work, it should pop up for you to download this automatically - but if not you will need to browse to the extension store to download this."